compiler/damlc/tests/daml-test-files/Interface.EXPECTED.md

```diff
@@ -20,24 +20,28 @@
+> : [Implements](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-interface-implements-77034) t Token =\> t -\> [Party](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-party-50311) -\> [Update](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-update-36457) ([ContractId](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-contractid-47171) Token)
 
 <a name="function-interface-splitimpl-48531"></a>[splitImpl](#function-interface-splitimpl-48531)
 
-> : Implements t Token =\> t -\> [Int](https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-68728) -\> [Update](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-update-36457) ([ContractId](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-contractid-47171) Token, [ContractId](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-contractid-47171) Token)
+> : [Implements](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-interface-implements-77034) t Token =\> t -\> [Int](https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-68728) -\> [Update](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-update-36457) ([ContractId](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-contractid-47171) Token, [ContractId](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-contractid-47171) Token)
 
 <a name="function-interface-setamount-71357"></a>[setAmount](#function-interface-setamount-71357)
 
-> : Implements t Token =\> t -\> [Int](https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-68728) -\> Token
+> : [Implements](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-interface-implements-77034) t Token =\> t -\> [Int](https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-68728) -\> Token
 
 <a name="function-interface-getamount-93321"></a>[getAmount](#function-interface-getamount-93321)
 
-> : Implements t Token =\> t -\> [Int](https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-68728)
+> : [Implements](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-interface-implements-77034) t Token =\> t -\> [Int](https://docs.daml.com/daml/stdlib/Prelude.html#type-ghc-types-int-68728)
 
 <a name="function-interface-getowner-9315"></a>[getOwner](#function-interface-getowner-9315)
 
-> : Implements t Token =\> t -\> [Party](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-party-50311)
+> : [Implements](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-interface-implements-77034) t Token =\> t -\> [Party](https://docs.daml.com/daml/stdlib/Prelude.html#type-da-internal-lf-party-50311)
+
+<a name="function-interface-dollarimplementsassettoken-85354"></a>[($implements\_Asset\_Token)](#function-interface-dollarimplementsassettoken-85354)
```

Comment:
we probably want to filter these somehow?

Reply:
yeah, in `DA.Daml.Doc.Extract` in `getFctDocs` there the following line to filter out `_choice_`. You can add another line:

```
    guard (not $ "_choice_" `T.isPrefixOf` packRdrName name)
```

Reply:
thanks!

Reply:
Done now, thanks again!